dbt_subprojects/daily_spellbook/models/_projects/gmx/event/arbitrum/gmx_v2_arbitrum_sources.yml

```diff
@@ -0,0 +1,15 @@
+version: 2
+
+sources:
+  - name: gmx_v2_arbitrum
+    description: A decentralized, permissionless perpetual trading DEX on the Arbitrum blockchain, allowing peer-to-peer trading without reliance on oracles or governance.
+    tables:
+      - name: EventEmitter_evt_EventLog1
+      - name: EventEmitter_evt_EventLog2
+
+  - name: ai_data_master
+    description: Materialized view that extracts token information from the Arbitrum API, including symbol, contract address, decimals, and synthetic status.
+    tables:
+      - name: result_token_information_from_arbitrum_api  
+        meta:
+          materialized: view          
```

Comment:
```suggestion

```

Reply:
no need for this, handle that in model itself

Reply:
Done